Add an example for nested json parsing (#5376)
DataDog/documentation: content/en/logs/processing/parsing.md

{{< img src="logs/processing/parsing/parsing_example_5_bis.png" alt="Parsing example 5 bis" responsive="true" style="width:80%;" >}}

### Nested JSON

> You might have logs that contains JSON nested after a raw prefix. In those cases, you can use the `json` **filter** to easily parse this JSON format.

Reviewer: ```suggestion
Use the `json` *filter* to parse a JSON object nested after a raw text prefix:
```

**Log**: 

```
Sep 06 09:13:38 vagrant program[123]: server.1 {"method":"GET","status_code":200,"url":"https://app.datadoghq.com/logs/pipelines","duration":123456}
```

**Rule**:

```

> myparsingrule %{date("MMM dd HH:mm:ss"):timestamp} %{word:vm} %{word:app}\[%{number:logger.thread_id}\]: %{notSpace:server} %{data::json}

Reviewer: ```suggestion
parsing_rule %{date("MMM dd HH:mm:ss"):timestamp} %{word:vm} %{word:app}\[%{number:logger.thread_id}\]: %{notSpace:server} %{data::json}
```
